### ISSUE TITLE
Option to exclude latest commit from changelog

### ISSUE THREAD

**Issue by silverwind** [BigWigsMods/packager#146]
I use other tools to create a release commit so the commit history looks like:

```
- commit 3: 1.0.0
- commit 2: fix2
- commit 1: fix1
```

package will generate a changelog with a duplicate version entry for it:

```
1.0.0
- 1.0.0
- fix 2
- fix 1
```

It'd be nice to have an option to exclude the latest commit to remove this duplicate "1.0.0" changelog entry.

**Comment by nebularg** [comment]
10 different devs, 10 different changelog preferences 😛 Which is why you can build it yourself and set the `manual-changelog` in your pkgmeta

**Comment by silverwind** [comment]
Does the `manual-changelog` option expect only the latest version's changelog in the file, e.g. should I reset it every release?

**Comment by nebularg** [comment]
Yup, the file you specify is treated just like the generated changelog, ie, it's attached to each uploaded file. So changes since the last tag (like what the packager does) is what I'd recommend

**Comment by silverwind** [comment]
[Implemented it](https://github.com/silverwind/idTip/commit/6d72923083ae6d2449db3803be74fc4587cb4ba8).
